sky/services/editing/editing.mojom

```diff
@@ -48,3 +48,13 @@ interface Keyboard {
   Show();
   Hide();
 };
+
+struct ClipData {
+  string text;
+};
+
+[ServiceName="editing::Clipboard"]
+interface Clipboard {
+  SetClipData(ClipData clip);
+  GetClipData() => (ClipData? clip);
```

Comment:
Using an enum means that the framework has to define all possible types. I can imagine some operations allowing for user-created type transfer where the sender and receiver agree on some uniquely identified format through the string argument, e.g. "com.example.album-schema-xml".  The framework could define string constants for certain well known types, e.g. text, image, URL, etc.